packages/nuxt/src/app/composables/loading-indicator.ts

```diff
@@ -61,7 +63,7 @@ function createLoadingIndicator (opts: Partial<LoadingIndicatorOpts> = {}) {
     if (at >= 100) { return finish() }
     clear()
     progress.value = at < 0 ? 0 : at
-    if (throttle && import.meta.client) {
+    if (throttle && import.meta.client && !immediate) {
```

Comment:
This options, just like `throttle = 0`?

Reply:
Yes, I thought it would be more explicit and cleaner than trying to update throttle